src/main/java/seedu/ui/UI.java

```diff
@@ -1,10 +1,13 @@
 package seedu.ui;
 
+import seedu.StudentList;
 import seedu.attendance.Attendance;
 import seedu.event.Event;
 import seedu.performance.Performance;
 import java.io.Serializable;
 import java.util.ArrayList;
+import java.util.HashMap;
```

Comment:
Where are you using HashMap?

Reply:
It doesn't seem used

Reply:
ahhhhh i was trying hashmap for studentlist and forgot to delete

Reply:
thanks for pointing it out!